engine/time/src/main/java/io/deephaven/time/calendar/StaticCalendarMethods.java

```diff
@@ -1,414 +1,435 @@
 /**
- * Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+ * Copyright (c) 2016-2023 Deephaven Data Labs and Patent Pending
  */
+/****************************************************************************************************************************
+ ****** AUTO-GENERATED CLASS - DO NOT EDIT MANUALLY - Run StaticCalendarMethodsGenerator or "./gradlew :Generators:generateStaticCalendarMethods" to regenerate
+ ****************************************************************************************************************************/
+
 package io.deephaven.time.calendar;
 
+import io.deephaven.time.calendar.BusinessCalendar;
+import io.deephaven.time.calendar.BusinessDay;
+import io.deephaven.time.calendar.Calendar;
+import java.lang.String;
 import java.time.DayOfWeek;
 import java.time.Instant;
 import java.time.LocalDate;
+import java.time.LocalTime;
 import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.Map;
+import java.util.Set;
```

Comment:
These are from generated code.  The generator is making certain that input/output types are included in case they are used in the function bodies.  If you find it particularly objectionable, I can remove it, but it isn't hand written code.